local-cli/start_packager.sh

```diff
@@ -0,0 +1,22 @@
+
+set -e
+
+HEADER="${SRCROOT}/Modules/RCTIPAddress.h"
+
+echo "// This file is generated by a build script." > $HEADER
+echo "// Updated on "`date` >> $HEADER
+echo "" >> $HEADER
+
+ip_address="`ifconfig en0 | grep "inet " | cut -d: -f2 | awk '{print $2}'`"
```

Comment:
@frantic has a good point that this might not work for all cases.

I believe new macs that are only wireless use `en0` however, older macs that use `en0` when wired in and `en1` for wifi.